packages/contracts-bedrock/interfaces/dispute/IDisputeGame.sol

```diff
@@ -13,6 +13,7 @@ interface IDisputeGame is IInitializable {
     function gameType() external view returns (GameType gameType_);
     function gameCreator() external pure returns (address creator_);
     function rootClaim() external pure returns (Claim rootClaim_);
+    function rootClaimByChainId(uint256 _chainId) external view returns (Claim rootClaim_);
```

Comment:
nit, for consistency with `rootClaim` as it'll be odd if it needed a different modifier.
```suggestion
    function rootClaimByChainId(uint256 _chainId) external pure returns (Claim rootClaim_);
```